bugzoo/mgr/container.py

```diff
@@ -50,6 +50,16 @@ def clear(self) -> None:
         Closes all running containers.
         """
         logger.debug("clearing all running containers")
+        try:
+            all_uids = [uid for uid in self.__containers.keys()]
+            for uid in all_uids:
+                try:
+                    self.__delitem__(uid)
+                except KeyError:
+                    # Already deleted
+                    pass
+        except AttributeError:
+            logger.debug("containers do not exist yet.")
         self.__containers = {}
```

Comment:
All of these lines below should be redundant.

Reply:
Ah, I see. Maybe we shouldn't use `clear` to initialise? Seems like a bit of a code smell.

Reply:
Yeah, seems like a bad idea